src/newLaunch/baseStage.scss

```diff
@@ -101,9 +134,18 @@
   background-color: rgba(64, 52, 83, 1);
   color: $White;
   cursor: pointer;
+}
+
+.make-admin-button {
```

Comment:
I suggest we follow only one naming convention. I guess camelCase is should be used across `prime-launch`.

Reply:
there seems to be a mix of naming conventions used - kebab case, some bem naming, camel case. this change was introduced in july
![image](https://user-images.githubusercontent.com/85691163/190571953-6da6eb47-f508-49b8-9307-05eff1d1da0c.png)

this specific class `.make-admin-button` was also introduced by someone else, also in july. my point is, your request would entail a refactor, which we do not want to do right now, correct ?

Reply:
Make sense. Probably should be done together (or alongside) the formatting task you've suggested in the other comment...

Reply:
:+1: